README.md

```diff
@@ -163,3 +163,65 @@ Interested in Integrating SocialConnect, get in touch by filling this [form](htt
 ## 📣 Feedback
 
 **SocialConnect is in beta**! Help us improve by sharing feedback on your experience in the Github Discussion section. You can also open an issue or a PR directly on this repo.
+
+## FAQ
+
+<details>
+  <summary>What is an Identifier?</summary>
+
+Identifier is any string of text that a user can use to identify other user.
+
+For example, Phone number, Twitter handle, GitHub username etc...
+
+</details>
+
+<details>
+  <summary>What is an Issuer?</summary>
+
+Issuer is an entity that is willing to take the overhead of verifying a user's ownership of an identifier.
+
+</details>
+
+<details>
+  <summary>Does Issuer need to pay for gas?</summary>
+
+Yes, Issuer pays for gas every time it wants to attest a new user.
```

Comment:
this isn't strictly true--the issuer only pays for gas if they publish the attestations on-chain on behalf of the user. Issuers can return the signed attestation to the user, who can then choose to publish this to `FederatedAttestations.sol` or not.